src/ui/mystery-encounter-ui-handler.ts

```diff
@@ -506,31 +506,33 @@ export default class MysteryEncounterUiHandler extends UiHandler {
 
     // Sets up the mask that hides the description text to give an illusion of scrolling
     const descriptionTextMaskRect = globalScene.make.graphics({});
+    const MaskHeight = 57;
```

Comment:
```suggestion
    const maskHeight = 57;
```

Minor nitpick, this should be lowercase. I'll mark the other uses of it so you can commit them as a batch